### ISSUE TITLE
[tgi][lynx] TGI_COLOR_BLACK is transparent and not black color

### ISSUE THREAD

**Issue by Fabrizio-Caruso** [cc65/cc65#956]
(Problem only verified in the Atari Lynx target.)
All colors except TGI_COLOR_BLACK, when used as background color, are displayed on screen (and can possibly erase whatever was below). This is not the case for TGI_COLOR_BLACK, as you can verify with the following simple program:
```c
#include "display_macros.h"
#include "graphics_settings.h"
#include <tgi.h>
#define CLI() asm("\tcli")
int main(void)
{
    tgi_install (tgi_static_stddrv);
    tgi_init ();        
    CLI();
    while (tgi_busy())  {  };
    tgi_setpalette(tgi_getdefpalette());
    tgi_setbgcolor(TGI_COLOR_GREY);
    tgi_setcolor(TGI_COLOR_RED);
    tgi_outtextxy(8,8,"hello"); 
    tgi_outtextxy(8,8*4,"hello");
    tgi_setbgcolor(TGI_COLOR_WHITE);
    tgi_outtextxy(8,8,"    "); 
    tgi_setbgcolor(TGI_COLOR_BLACK);
    tgi_outtextxy(8,8*4,"    ");
    while(1){};
    return 0;
}
```
See attached picture to verify problem:
![image](https://user-images.githubusercontent.com/29057615/66564630-beae6a00-eb60-11e9-9230-024ea6762100.png)

**Comment by greg-king5** [comment]
Fixed by commit f3f15cfd25dc5379ca48021884f4b96471dd895e.

**Comment by Fabrizio-Caruso** [comment]
@greg-king5 Thanks a lot for the fix!
@Philsan69 has validated the fix on real hardware.
